tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "noImplicitAny": true,
     "strictBindCallApply": true,
     "strictNullChecks": true,
+    "skipLibCheck": true,
```

Comment:
Can you create a follow up issue to enable this again and assign it to me? I don't want to forget about it.

Reply:
https://github.com/open-formulieren/formio-builder/issues/211